src/components/[guild]/Onboarding/components/SummonMembers/SummonMembers.tsx

```diff
@@ -66,7 +75,12 @@ const SummonMembers = () => {
             <PulseMarker colorScheme="red" placement="top">
               <Button
                 h="10"
-                onClick={onOpen}
+                onClick={() => {
+                  captureEvent(
+                    "guild creation flow > click on 'Send Discord join button'"
```

Comment:
Please remove the `'`s (both here and in other places), just to make sure it's easy to query these events in PostHog